commands/utils/utils.go

```diff
@@ -64,3 +67,23 @@ func Md5Hash(values ...string) (string, error) {
 	}
 	return hex.EncodeToString(hash.Sum(nil)), nil
 }
+
+func UploadScanToGitProvider(scanResults []services.ScanResponse, params *FrogbotParams, client vcsclient.VcsClient) error {
+	// don't do anything if scanResults is empty
```

Comment:
Shouldn't we add it a test to make sure it is GitHub? Something like:
```
if git provider != GitHub {
    return
}
```

Reply:
done